includes/discovery/libvirt-vminfo.inc.php

```diff
@@ -66,10 +66,34 @@
                     $vmwVmDisplayName = $xml->name;
                     $vmwVmGuestOS     = '';
                     // libvirt does not supply this
-                    $vmwVmMemSize = ($xml->currentMemory / 1024);
-                    exec($config['virsh'].' -c '.$uri.' domstate '.$dom_id, $vm_state);
+                    exec($config['virsh'].' -rc '.$uri.' domstate '.$dom_id, $vm_state);
                     $vmwVmState = ucfirst($vm_state[0]);
-                    $vmwVmCpus  = $xml->vcpu;
+
+                    $vmwVmCpus = null;
+                    $vmwVmMemSize = null;
+                    if ($xml->xpath("//sysinfo/system/entry[@name='manufacturer'][.='oVirt']")) {
+                        d_echo("Parsing information for $vmwVmDisplayName with oVirt quirx");
```

Comment:
Not sure what you're referring to with quirkx but the call for help is testing non-oVirt. oVirt has been tested with this pull request.